sql/migrations/20220705212140_world.sql

```diff
@@ -0,0 +1,122 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20220705212140');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20220705212140');
+-- Add your query below.
+
+
+-- Correct position of Raven Hill Spirit Healer
+UPDATE `creature` SET  `position_x` = -10599.2, `position_y` = 294.208, `position_z` = 31.7003, `orientation` = 3.14159 WHERE `guid` = 12783;
+-- Correct position of Crown Guard Tower Spirit Healer
+UPDATE `creature` SET  `position_x` = 1978.22, `position_y` = -3666.04, `position_z` = 119.364, `orientation` = 1.36136 WHERE `guid` = 9386;
+-- Correct position of Scarab Wall Spirit Healer
+UPDATE `creature` SET  `position_x` = -7988.49, `position_y` = 1557.96, `position_z` = 4.73619, `orientation` = 3.10669 WHERE `guid` = 7716;
+-- Correct position of Booty Bay Spirit Healer
+UPDATE `creature` SET  `position_x` = -14280.6, `position_y` = 300.899, `position_z` = 31.2034, `orientation` = 4.50295 WHERE `guid` = 40519;
+-- Correct position of Flame Crest Spirit Healer
+UPDATE `creature` SET  `position_x` = -7493.53, `position_y` = -2115.28, `position_z` = 136.081, `orientation` = 4.7473 WHERE `guid` = 40528;
+-- Correct position of Kargath Spirit Healer
+UPDATE `creature` SET  `position_x` = -6811.41, `position_y` = -2284.96, `position_z` = 280.836, `orientation` = 5.88176 WHERE `guid` = 40529;
+-- Correct position of Thelsamar Spirit Healer
+UPDATE `creature` SET  `position_x` = -5351.07, `position_y` = -2890.37, `position_z` = 341.881, `orientation` = 1.7693 WHERE `guid` = 40533;
+-- Correct position of Baradin Bay Spirit Healer
+UPDATE `creature` SET  `position_x` = -3341.49, `position_y` = -874.43, `position_z` = 1.05191, `orientation` = 1.93732 WHERE `guid` = 40535;
+-- Correct position of Southshore Spirit Healer
```

Comment:
This has been pasted twice by mistake. This Spirit Healer is in Arathi Highlands.

Reply:
My bad. It's just the comment that is bad. The actual query changes the Arathi Highlands spirit healer.